### DISCUSSION TITLE
Welcome to datacap Discussions!

### DISCUSSION THREAD

**Discussion by qianmoQ** [devlive-community/datacap#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lzh-space** [comment]
![image](https://github.com/devlive-community/datacap/assets/4439517/e4faea4f-2b03-49fc-a12f-a18c67cdeba0)
如果对某一个人设置多个权限，菜单显示会有重复性问题

**Reply by qianmoQ** [reply]
这是一个 bug，已经转换为 issues

**Reply by qianmoQ** [reply]
已经修复

**Comment by lzh-space** [comment]
![image](https://github.com/devlive-community/datacap/assets/4439517/0d5fa33b-0c5c-446c-a880-35bfa8fef73c)
新增仪表盘后，建议按文档上有预览功能。目前最新版本上是没有这个预览功能。

**Reply by qianmoQ** [reply]
新版的点击仪表盘名字即可
